.travis.yml

```diff
@@ -15,8 +15,10 @@ jobs:
   - stage: Verify
     name: Lint
     script: npm run lint
-  - script: npm run test
-    name: Test
+  - stage: Test
+    script: npm run travis:verify
```

Comment:
I think we can make this just `
curl -sSL "https://raw.githubusercontent.com/RedHatInsights/insights-interact-tools/refs/heads/main/scripts/coverage.sh" | bash` and also get rid of the `travis:verify`, but we should doublecheck that this task isn't used in any other CI pipeline.

Reply:
I'm thinking we could achieve the same effect by instead changing the value of test:coverage to this curl? That's where the coverage script is referenced anyway, and potential pipelines depending on travis will then not need to change anything

Reply:
Sounds good! I was just thinking that the `yml` file might be more convenient with the length of the string and escaping quotes.